### PR TITLE
Added how-to for permissions issues with upboard

### PR DESCRIPTION
**Pull request by kohlerj**

### CODE REVIEW

**Comment by kohlerj** [comment]
@foehnph Merci d'accepter!!! ;)

File: README.md [hunk 1]
```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there is only a few points you need to check if you have issue with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the SBUS bridge)
```
Comment (mfaessle) [inline README.md]: "there are"
"issues"
Can you link to the SBUS bridge since it is in a different repository?

File: README.md [hunk 2]
```diff
@@ -40,7 +40,15 @@ TODO
...
+
+1.  You are using an "upboard" kernel. Check for that using `uname -a`
```
Comment (mfaessle) [inline README.md]: How is one supposed to see from this output if he runs an upboard kernel?

File: README.md [hunk 3]
```diff
@@ -40,7 +40,15 @@ TODO
...
+      -  If not, go this page and install such a kernel: [https://wiki.up-community.org/Ubuntu](https://wiki.up-community.org/Ubuntu)
+   
+2.  You have set the permissions and installed the add-ons as described here: [https://wiki.up-community.org/Ubuntu#Enable_the_HAT_functionality_from_userspace](https://wiki.up-community.org/Ubuntu#Enable_the_HAT_functionality_from_userspace)
+
+3.  You are using the wrong port name which is usually `/dev/ttyS1` or `/dev/ttyS4`.
```
Comment (mfaessle) [inline README.md]: This could be confusing whether the example ports are the wrong or right ones

File: README.md [hunk 4]
```diff
@@ -40,7 +40,15 @@ TODO
...
+     -   Make sure that you don't overwrite the parameters you set. For the SBUS bridge for instance, the value is set in the parameters `.yaml` file but it could be overwritten in another parameter file or launch file. Thus, check the ROS info message during when you launch the node. Note that `/dev/ttyS0` is reserved for the console, so try `cat /dev/ttySX` and try the ports which do not output any error (don't use `sudo`)
```
Comment (mfaessle) [inline README.md]: remove "during"
...try "using" the ports...

File: README.md (hunk 1 above)
Comment (kohlerj) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (kohlerj) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (kohlerj) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (kohlerj) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there are only a few points you need to check if you have issues with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the [SBUS bridge](https://github.com/uzh-rpg/rpg_quadrotor_control) )
+
+1.  You are using an "upboard" kernel. Check for that using `uname -a` and look for any mention of "upboard" in the kernel's name.
+      -  If not, go this page and install such a kernel: [https://wiki.up-community.org/Ubuntu](https://wiki.up-community.org/Ubuntu)
```
Comment (mfaessle) [inline README.md]: go "to"

File: README.md [hunk 6]
```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there are only a few points you need to check if you have issues with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the [SBUS bridge](https://github.com/uzh-rpg/rpg_quadrotor_control) )
```
Comment (mfaessle) [inline README.md]: I would make this link more specific: -> https://github.com/uzh-rpg/rpg_quadrotor_control/tree/master/bridges/sbus_bridge

**Review by mfaessle — APPROVED** [review]
scolgtm

**Pull request merged**